pkg/subnet/local.go

```diff
@@ -697,11 +724,33 @@ func getExpectedDefaultSnapshotSHA256Sum(isSingleNode bool) (string, error) {
 
 // Initialize default snapshot with bootstrap snapshot archive
 // If force flag is set to true, overwrite the default snapshot if it exists
-func SetDefaultSnapshot(snapshotsDir string, forceReset bool, isSingleNode bool) (bool, error) {
-	bootstrapSnapshotArchivePath := filepath.Join(snapshotsDir, constants.BootstrapSnapshotArchiveName)
-	if isSingleNode {
-		bootstrapSnapshotArchivePath = filepath.Join(snapshotsDir, constants.BootstrapSnapshotSingleNodeArchiveName)
+func SetDefaultSnapshot(snapshotsDir string, forceReset bool, avagoVersion string, isSingleNode bool) (bool, error) {
+	resetCurrentSnapshot := forceReset
+	isPreCortina17 := false
```

Comment:
nit: `bool` is `false` by default, so it can be just `var isPreCortina17 bool` but I am not sure which way is better

Reply:
I prefer the current one (I prefer explicit initialization for readability), but using the default init a convention so I wil change